index.js

```diff
@@ -0,0 +1,29 @@
+document.getElementById('hamburger').onclick = () => {
+  document.getElementById('navulitems').classList.add('hamburger');
+  document.getElementById('hamburger').classList.add('hamx');
+  document.getElementById('x').classList.remove('hamx');
+};
+
+document.getElementById('x').onclick = () => {
+  document.getElementById('navulitems').classList.remove('hamburger');
+  document.getElementById('hamburger').classList.remove('hamx');
+  document.getElementById('x').classList.add('hamx');
+};
+
+document.getElementById('l1').onclick = () => {
+  document.getElementById('navulitems').classList.remove('hamburger');
+  document.getElementById('hamburger').classList.remove('hamx');
+  document.getElementById('x').classList.add('hamx');
+};
+
+document.getElementById('l2').onclick = () => {
+  document.getElementById('navulitems').classList.remove('hamburger');
+  document.getElementById('hamburger').classList.remove('hamx');
+  document.getElementById('x').classList.add('hamx');
+};
+
+document.getElementById('l3').onclick = () => {
+  document.getElementById('navulitems').classList.remove('hamburger');
+  document.getElementById('hamburger').classList.remove('hamx');
+  document.getElementById('x').classList.add('hamx');
+};
```

Comment:
- [ ] As requested by the previous reviewer, your navbar should take the height of 100vh so whenever the user clicks the hamburger buttons it should take the whole height and the position of the navbar should be fixed. Please fix the opacity as in the Figma design. See the image below for reference
### your design

![your](https://user-images.githubusercontent.com/81727988/210551356-e54c1ae6-b8cd-4679-a234-b61bb35db123.png)

### Figma design
![fimga](https://user-images.githubusercontent.com/81727988/210551379-b5b6f1c9-c67f-4b47-9a29-9e0d58dc9e58.png)